app/controllers/carts_controller.rb

```diff
@@ -13,24 +13,29 @@ def create
   end
 
   def update
-    @course_unpaid.update_all(paid: true)
+    @course_unpaid.ids.map {|n|
+      @cart = Cart.find_by id: n
+      @course = Course.find_by(id: @cart.course_id)
+      @cart.update_attributes(paid: true)
```

Comment:
update 1 cart thôi mà liên quan hết đến @course_unpaid à em

Reply:
Là cart của current_user anh ạ. Trong cart của em vì sẽ không xóa những cart đã thanh toán nên phần tính tiền chỉ lấy những cart unpaid thôi ạ. Nên em thấy dù chỉ có 1 course được mua thì vẫn cần update unpaid ạ